### ISSUE TITLE
Windows 7 don't redraw properly

### ISSUE THREAD

**Issue by donandren** [AvaloniaUI/Avalonia#2247]
WIndows 7 don't redraw properly changes in controls state (mostly after popup is open/closed, but not limited to this scenario). UI is correctly redrawn only after:
- window minimize/restore
- resize.
- switch to different application and go back

Able to reproduce it very consistently when windows 7 is with non Aero Theme (e.g. classic), but also on some computers with Aero theme enabled.
Same issue exists on Immediate and Deferred renderer, Skia and Direct2D.
Tested on a VMware machine and real machine with Windows 7 all updated and video drivers up to date.
example:
- change item in dropdown view, item is changed, but not redrawn 
![2019-01-16_16-35-42](https://user-images.githubusercontent.com/13854631/51256332-db40bb80-19ad-11e9-9d02-3e9cfb8d89a0.gif)

**Comment by donandren** [comment]
For ImmediateRenderer there is a possible solution (not sure it's the most appropriate one), but hope it would help
at Invalidate:
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Windows/Avalonia.Win32/WindowImpl.cs#L286-L298

if we change the code to invalidate full area, problem is solved for win7:

```C#
...
        [DllImport("user32.dll")]
        private static extern bool InvalidateRect(IntPtr hWnd, IntPtr lpRect, bool bErase);

        public void Invalidate(Rect rect)
        {
             InvalidateRect(_hwnd, IntPtr.Zero, false);
        }
...
```

**Comment by Flithor** [comment]
@maxkatz6 I've found it still exists on V0.10.21.
It has been fixed in V0.11? Will it be fixed in V0.10?

**Comment by timunie** [comment]
@Flithor 
- Win7 is not supported by MS anymore, so it's not official supported by Avalonia. It still somewhat works, but there is no guarantee on it. 
- 0.10.x will get some critical bug fixes, but most of the stuff is only going into v11. Especially Immediate and Differed-Renderer will not be maintained further as we have a way better Composition-Renderer now.

**Comment by maxkatz6** [comment]
@Flithor no
